packages/cli-platform-apple/src/tools/pods.ts

```diff
@@ -61,10 +60,26 @@ export function generateMd5Hash(text: string) {
   return createHash('md5').update(text).digest('hex');
```

Comment:
Hello @szymonrybczak ,

I don’t claim to be right, but it might be better to update the `MD5` algorithm to `SHA-256`.  If I’m wrong, could you please explain why?

Reply:
We can but I don't think there are benefits of doing so? It's relatively simple scenario.